cub/benchmarks/bench/transform/other.cu

```diff
@@ -0,0 +1,194 @@
+// SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+// %RANGE% TUNE_THREADS tpb 128:1024:128
+// %RANGE% TUNE_ALGORITHM alg 0:1:1
+
+// keep checks at the top so compilation of discarded variants fails really fast
+#if !TUNE_BASE
```

Comment:
Nitpick: Can we find a better name than `other`

Reply:
I honestly still need to grasp whether I should just split each benchmark into a dedicated file. Then those would have better names.

Reply:
I split the benchmarks and gave them better names.